lib/chef/knife/vsphere_vm_network_add.rb

```diff
@@ -0,0 +1,64 @@
+require 'chef/knife'
```

Comment:
according to 
[http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/FrozenStringLiteralComment](url)
just add 
<pre>
# frozen_string_literal: true
</pre>
to the top of the file